src/components/map/layers/NetworkLayers.tsx

```diff
@@ -236,6 +238,11 @@ class NetworkLayers extends Component<INetworkLayersProps> {
         return { className: s.hidden };
     }
 
+    private recreateReaction = (type: GeoserverLayer) => (reaction: IReactionDisposer) => {
```

Comment:
setVectorgridLayerReaction?

I would add a comment too:
// Sets a reaction object for GeoserverLayer (replaces existing one) so that reaction object's wouldn't multiply each time a VectorGridLayer is re-rendered.